bitrise.yml

```diff
@@ -92,6 +96,45 @@ workflows:
         - podfile_path: $PODFILE_PTH
         - verbose: false
 
+  test_with_specific_ruby:
+    envs:
+    # Podfile is in repo root
+    - REPO_URL: https://github.com/bitrise-samples/ios-cocoapods-1.x.git
+    - TEST_REPO_BRANCH: master
+    - PODFILE_PTH: ''
+    steps:
+    - change-workdir:
+        inputs:
+        - path: "${ORIG_BITRISE_SOURCE_DIR}"
+        - is_create_path: "false"
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            rm -rf ./_tmp
+
+            git clone "${REPO_URL}" ./_tmp --branch "${TEST_REPO_BRANCH}"
+    - change-workdir:
+        title: Switch working dir to test/_tmp dir
+        description: |-
+          To prevent step testing issues, like referencing relative
+          files with just './some-file', which would work for local tests
+          but not if the step is included in another bitrise.yml!
+        inputs:
+        - path: "${ORIG_BITRISE_SOURCE_DIR}/_tmp"
+        - is_create_path: "false"
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+
+            echo "2.6.0" >> .ruby-version
+    - path::./:
```

Comment:
Shouldn't we check after this that the Ruby was installed with a script step?

Reply:
We don't need, because if the step fails to install, then the pod install will fail because of the invalid ruby version.